src/PublishPort.jsx

```diff
@@ -20,14 +20,14 @@ export const PublishPort = ({ id, item, onChange, idx, removeitem, itemCount })
                     <Popover aria-label={_("IP address help")}
                         enableFlip
                         bodyContent={_("If host IP is set to 0.0.0.0 or not set at all, the port will be bound on all IPs on the host.")}>
-                        <button onClick={e => e.preventDefault()} className="pf-c-form__group-label-help">
+                        <button onClick={e => e.preventDefault()} className="pf-v5-c-form__group-label-help">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1322-20230703-044046-64a79ae1-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)